### PR TITLE
Update docfx.json

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnet/docfx.json [hunk 1]
```diff
@@ -72,13 +72,13 @@
...
-      "feedback_system": "GitHub",
-      "feedback_github_repo": "MicrosoftDocs/docs-help-pr",      
-      "searchScope": ["help-docs"],
-      "recommendations": true
-    }
+    "markdownEngineName": "markdig",
+    "globalMetadata": {
+          "breadcrumb_path": "/aspnet/breadcrumb/toc.json",
+          "feedback_system": "GitHub",
+          "feedback_github_repo": "MicrosoftDocs/docs-help",
```
Comment (gewarren) [inline aspnet/docfx.json:79]:
```suggestion
          "feedback_github_repo": "dotnet/aspnetdocs",
```

File: aspnet/docfx.json [hunk 2]
```diff
@@ -72,13 +72,13 @@
...
+          "searchScope": ["help-docs"],
```
Comment (gewarren) [inline aspnet/docfx.json:80]: You already have searchScope set to ASP.NET on line 46.

**Review by gewarren — COMMENTED** [review]
I'm still confused why you want two separate globalMetadata sections. Makes the file a bit harder to follow?

**Review by gewarren — APPROVED** [review]
The alignment got messed up but otherwise looks good to me!

File: aspnet/docfx.json [hunk 3]
```diff
@@ -36,9 +36,10 @@
     "overwrite": [],
     "externalReference": [],
     "globalMetadata": {
-      "_navPath": "/foo",
+            "_navPath": "/foo",
       "_navRel": "/foo",
       "breadcrumb_path": "/aspnet/breadcrumb/toc.json",
+      "recommendations": true,
```
Comment (Rick-Anderson) [inline aspnet/docfx.json]: This is the only change from the pre- recommendations version. All the rest is white space the editor might have added.

File: aspnet/docfx.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "build": {
+    "build": {
```
Comment (gewarren) [inline aspnet/docfx.json:2]:
```suggestion
  "build": {
```

File: aspnet/docfx.json [hunk 5]
```diff
@@ -39,6 +39,7 @@
       "_navPath": "/foo",
       "_navRel": "/foo",
       "breadcrumb_path": "/aspnet/breadcrumb/toc.json",
+      "recommendations": true
```
Comment (gewarren) [inline aspnet/docfx.json:42]:
```suggestion
      "recommendations": true,
```

File: aspnet/docfx.json [hunk 6]
```diff
@@ -72,13 +73,6 @@
...
-    "markdownEngineName": "markdig"
-  },
-"globalMetadata": {
-      "breadcrumb_path": "/aspnet/breadcrumb/toc.json",
-      "feedback_system": "GitHub",
-      "feedback_github_repo": "MicrosoftDocs/docs-help-pr",      
-      "searchScope": ["help-docs"],
-      "recommendations": true
-    }
+    "markdownEngineName": "markdig",
```
Comment (gewarren) [inline aspnet/docfx.json:76]:
```suggestion
    "markdownEngineName": "markdig"
```

**Review by gewarren — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@gewarren  see https://docs.microsoft.com/en-us/aspnet/mvc/overview/getting-started/introduction/adding-a-controller and other docs, no suggestions.

**Pull request merged**